shared/helpers/puppeteer.js

```diff
@@ -163,6 +222,7 @@ async function goTo(page, path) {
     throw new Error(`HTTP ${code} for '${pathname}'`)
```

Comment:
**question** That's something for a separate PR, I think, but I'm wondering if by not hitting the review app, we're not undoing the work from [this PR](https://github.com/alphagov/govuk-frontend/pull/3963).

Reply:
We're using the same `goToComponent()` method still which runs `slug()`

So all the URL escaping is still there, we just mock the response now:

```mjs
const exampleName = slug(options?.exampleName ?? '', { lower: true })
```

Reply:
Don't forget too it's only the `/tests/boilerplate` route + `#slot` that this PR removes

All the component previews are still there

Reply:
Cool! That wasn't blocking, just wanted to flag it so we don't forget that at the moment `goTo` was both used for testing our components, as well as checking that the review app responded to the right URLs for their preview. But you likely had it in mind anyway given you did the work to add that check for 400 statused 😊